homeassistant/components/sensor/geizhals.py

```diff
@@ -80,6 +80,8 @@ def state(self):
     @property
     def device_state_attributes(self):
         """Return the state attributes."""
+        while (len(self.data.prices) < 4):
+            self.data.prices.append("None")
```

Comment:
I think it is better if the price count is fixed, because if you create a template for a price3 and at some moment in time geizhals does not have a price for price3 we will have issue in template.
If you think that a template can handle this situation, then your suggestion is acceptable.